spec/requests/general_info_spec.rb

```diff
@@ -40,6 +40,16 @@ def expect_form_failure
     expect_form_success
   end
 
+  it "doesn't allow incidents with years that have already been submitted for" do
+    AgencyStatus.find_or_create_by_ori(User.first.ori)
+                .mark_year_submitted!(Time.current.year - 1)
+
+    answer_all_general_info
```

Comment:
Let's explicitly pass in the current year, so we aren't using an implied behavior of a library function